src/Utility/Xml.php

```
@@ -150,6 +150,8 @@ protected static function _loadXml($input, $options)
                 $xml = new DOMDocument();
                 $xml->loadXML($input);
             }
+        } catch (\Throwable $e) {
+            $xml = null;
         } catch (Exception $e) {
             $xml = null;
```

anyone know why this silently fails to null?